viewer/test/test-add-heading-anchors.js

```diff
@@ -65,6 +65,14 @@ describe("addHeadingAnchors", function () {
 
       Array.prototype.forEach.call(this.headingsWithoutAnId, assertNoAnchorAdded);
     });
+
+    it("does not add more anchors if init is called more than once", function () {
+      var count = this.testcontainer.querySelectorAll("a.headerLink").length;
```

Comment:
I slightly regret my decision to not camelcase testcontainer (because it's named after a css selector).  Oh well.  for now.